PWGHF/D2H/Tasks/taskCharmPolarisation.cxx

```diff
@@ -138,33 +146,34 @@ struct TaskPolarisationCharmHadrons {
     const AxisSpec thnAxisCosThetaStarBeam{configThnAxisCosThetaStarBeam, "cos(#vartheta_{beam})"};
     const AxisSpec thnAxisMlBkg{configThnAxisMlBkg, "ML bkg"};
     const AxisSpec thnAxisMlNonPrompt{configThnAxisMlNonPrompt, "ML non-prompt"};
+    const AxisSpec thnAxisIsRotatedCandidate{configThnAxisIsRotatedCandidate, "0: standard candidate, 1: rotated candidate"};
 
     if (doprocessDstarWithMl || doprocessLcToPKPiWithMl) {
       if (activateTHnSparseCosThStarHelicity) {
-        registry.add("hSparseCharmPolarisationHelicity", "THn for polarisation studies with cosThStar w.r.t. helicity axis and BDT scores", HistType::kTHnSparseF, {thnAxisInvMass, thnAxisPt, thnAxisPz, thnAxisY, thnAxisCosThetaStarHelicity, thnAxisMlBkg, thnAxisMlNonPrompt});
+        registry.add("hSparseCharmPolarisationHelicity", "THn for polarisation studies with cosThStar w.r.t. helicity axis and BDT scores", HistType::kTHnSparseF, {thnAxisInvMass, thnAxisPt, thnAxisPz, thnAxisY, thnAxisCosThetaStarHelicity, thnAxisMlBkg, thnAxisMlNonPrompt, thnAxisIsRotatedCandidate});
```

Comment:
Unless we want to implement the track rotation also for D* (which might be a good idea), I would add the additional axis only for Lc->pKpi

Reply:
I followed your suggestion, and I removed this further axis for the D*+. It required a simpe but a bit tedious re-coding of the several `if` statements. I re-tested for Lc locally and it works.

Reply:
then if needed we can add rot. bkg for D*+ as well, but since it does not seem strictly necessary at the moment I would avoid it